schemas/json/README.md

```diff
@@ -58,33 +58,37 @@ We explicitly forbid empty JSON arrays to avoid confusion about attributes which
 For example, ``submodelElements`` instead of ``submodelElement`` in case of [Submodel] class.
 
 If plural form made no sense for a property, we kept it as-is (*e.g.*, `isCaseOf`).
 The full list of exceptions is available [as code in aas-core-meta].
 
-If plural form made no sense for an attribute, we kept it as-is (*e.g.*, `isCaseOf`).
-The full list of exceptions is available [as code in aas-core-meta].
-
 [as code in aas-core-meta]: https://github.com/aas-core-works/aas-core-meta/blob/95055a55a8c8f60d75fb48c26eb932ff99945dd2/tests/test_v3rc2.py#L1122
 
 ### Primitive attribute values
 
-The UML specification uses xsd types. For the mapping of xsd types to JSON types please refer to [Part 2 of the series of the Asset Adminsistration Shell].
+The UML specification uses XSD types. For the mapping of XSD types to JSON types please refer to [Part 2 of the series of the Asset Adminsistration Shell].
 
-There are the following exceptions: For [Property]/value and [Range]/min and [Range]/min the type xs:string is used because the type is only known at run-time. Its type is declared in [Property]/valueType or [Range]/valueType, resp.
-Note: in valueOnly Format of [Part 2 of the series of the Asset Adminsistration Shell] value has the type as declared in Property/valueType.
+There are the following exceptions: For [Property]/value and [Range]/min and [Range]/max a JSON string is used 
+because the type is only known at run-time. Its type is declared in [Property]/valueType or [Range]/valueType, resp.
+Note: in valueOnly Format of [Part 2 of the series of the Asset Adminsistration Shell] value has the JSON type as 
+declared in Property/valueType taking the mapping of XSD ot JSON types into account (see [Part 2 of the series of the Asset Adminsistration Shell]).
 
 [Part 2 of the series of the Asset Adminsistration Shell]: https://industrialdigitaltwin.org/wp-content/uploads/2021/11/Details_of_the_Asset_Administration_Shell_Part_2_V1.pdf?__blob=publicationFile&v=10#page=81
 
 
-### Be aware of possible type mismatches between xsd and JSON types
+### Be aware of possible type mismatches between XSD and JSON types
 
-Type mappings are not trivial. Some sort of type mapping is always needed when using different programming languages and different serialization formats. One way to avoid some JSON type conversion problems would have been to just use [JSON strings] for all primitive attributes values. 
-However, besides loss of semantics and higher effort for programmers there also is the risk of JSON injection attacks. 
+Type mappings are not trivial. 
+Some sort of type mapping is always needed when using different programming languages and different serialization formats. 
+One way to avoid some JSON type conversion problems would have been to just use [JSON strings] for all primitive attributes values. 
+However, besides loss of semantics and higher effort for programmers there also is a higher risk of using unsanitized JSON input.
```

Comment:
Please see the previous comment. There is equal risk of JSON injenction whether you use only JSON strings _or_ JSON strings+numbers+booleans.

The effort for the programmer is _higher_ (in terms of execution branches, analyzing edge cases etc.) when you use JSON strings+numbers+booleans.

My suggestion is to remove this sentence altogether, and state the real reason which is legacy systems.

Reply:
P.S. perhaps a clarification of my comment. The value  is anyOf boolean, number and string. The schema can not enforce that it is a number if `valueType` is, say, `xs:double`. If you don't sanitise the input, the schema allows you to set `value` of a Property to a string even if the value type mandates a number. That finally opens doors for JSON injection.

Reply:
The JSON injection risk comes into play with ValueOnly Format. With Normal Output Format the value is xs:string. See exceptions from mappings.